Extensions/BitmapText/JsExtension.js

```diff
@@ -323,42 +341,42 @@ module.exports = {
 
     object
       .addAction(
-        'SetFontSize',
-        _('Font size'),
-        _('Modify the font size of a Bitmap text object.'),
-        _('the font size'),
+        'Scale',
+        _('Scale'),
+        _('Change the scale of a Bitmap text object, 1 by default.'),
+        _('the scale'),
         '',
-        'res/actions/font24.png',
-        'res/actions/font.png'
+        'res/actions/scale24.png',
+        'res/actions/scale.png'
       )
       .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
       .useStandardOperatorParameters('number')
       .getCodeExtraInformation()
-      .setFunctionName('setFontSize')
-      .setGetter('getFontSize');
+      .setFunctionName('setScale')
+      .setGetter('getScale');
 
     object
       .addAction(
-        'SetBitmapFontResourceName',
+        'BitmapFontName',
         _('Font'),
         _('Change the font used by the Bitmap text object.'),
         _('Set the font of _PARAM0_ to _PARAM1_'),
-        _('Style'),
+        '',
         'res/actions/font24.png',
         'res/actions/font.png'
       )
       .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
       .addParameter('font', _('Font resource'), '', false)
       .getCodeExtraInformation()
-      .setFunctionName('setFont');
+      .setFunctionName('setFont'); // TODO a test mais on change pas de la font mais la resource directement, mettre une nouvelle action pour changer l'atlas en plus.
```

Comment:
Yes, though it can be a single action, with one parameter taking the new resource name and the new atlas texture resource name (I recommend a single action because you don't really change one without changing the other - at least the action should make it intuitive that both are heavily linked).